recipe/conda_build_config.yaml

```diff
@@ -20,9 +20,9 @@ c_compiler:          # [osx]
 cxx_compiler:        # [osx]
   - clang_bootstrap  # [osx]
 
-# Because of cling requiring gxx 10.
-cxx_compiler_version:  # [linux]
-  - 10                 # [linux]
+macos_machine:                 # [osx]
+  - x86_64-apple-darwin13.4.0  # [osx and x86_64]
+  - arm64-apple-darwin20.0.0   # [osx and arm64]
```

Comment:
@anutosh491, do you know how to tell conda to build both arm and x86 builds. Currently, iiuc, it only builds in x86 mode.